<ui version="4.0" >
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>qsynth - A fluidsynth Qt GUI Interface.

   Copyright (C) 2003-2025, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qsynthAboutForm</class>
 <widget class="QDialog" name="qsynthAboutForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>About</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qsynth.qrc">:/images/about1.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>4</number>
   </property>
   <property name="spacing" >
    <number>4</number>
   </property>
   <item row="2" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>8</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="ClosePushButton" >
     <property name="text" >
      <string>&amp;Close</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QToolButton" name="AboutQtButton" >
     <property name="focusPolicy" >
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="toolTip" >
      <string>About Qt</string>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="qsynth.qrc" >:/images/qtlogo1.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>8</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="2" row="0" column="1" colspan="2" >
    <widget class="QTextBrowser" name="AboutTextView" >
     <property name="lineWidth" >
      <number>1</number>
     </property>
     <property name="undoRedoEnabled" >
      <bool>false</bool>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
     <property name="openExternalLinks" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="IconPixmapLabel" >
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="pixmap" >
      <pixmap resource="qsynth.qrc" >:/images/qsynth.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="margin" >
      <number>2</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4" />
 <tabstops>
  <tabstop>AboutTextView</tabstop>
  <tabstop>AboutQtButton</tabstop>
  <tabstop>ClosePushButton</tabstop>
 </tabstops>
 <resources>
  <include location="qsynth.qrc" />
 </resources>
 <connections/>
</ui>
